packages/react-native-builder-bob/src/utils/patchCodegenAndroidPackage.ts

```diff
@@ -89,7 +89,19 @@ export async function patchCodegenAndroidPackage(
     })
   );
 
-  await fs.rm(path.resolve(codegenAndroidPath, 'java/com/facebook'), {
-    recursive: true,
-  });
+  if (
+    await fs.pathExists(
+      path.resolve(codegenAndroidPath, 'java/com/facebook/react/viewmanagers')
+    )
```

Comment:
Should we check the `type` instead? What happens you change the `type` from `all`/`view` to module? Does that leave generated view files in the file system?

Reply:
@satya164 I just tested it and when the type is `modules`, codegen doesn't generate view specs at all.

Reply:
@atlj yeah but my question was about what happens after you change from one type to another. i checked the code and it should be fine since we delete everything https://github.com/callstack/react-native-builder-bob/blob/87b0032acd66b7b14a5b358117a3d1497c59baad/packages/react-native-builder-bob/src/targets/codegen.ts#L32